extensions/adobe/experience/decisioning/proposition-interaction-detail.schema.json

```diff
@@ -7,43 +7,57 @@
   ],
   "$id": "https://ns.adobe.com/experience/decisioning/proposition-interaction-details",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "title": "Experience Event - Proposition Interaction Detail",
+  "title": "Proposition Interaction Details",
   "type": "object",
   "meta:abstract": true,
   "meta:extensible": true,
-  "description": "A proposition interaction is used to link an experience event to a prior decision proposition. Each decision output has a unique event ID that can be sent back with an experience event. The decision proposition is assumed to have contributed to the experience event and is now associated with it.",
+  "description": "A proposition interaction is used to report an experience event that relates to a prior decision proposition. The decision proposition is assumed to have contributed to the experience and the unique decision ID is added to the experience event.",
   "definitions": {
     "proposition-interaction-detail-datatype": {
       "type": "object",
       "properties": {
+        "xdm:id": {
```

Comment:
This was mandated by all the clients of this use case. They wanted to use the payload from the API response and add it to the experience event. The id value of the proposition is kept in a property "id"